tests/phpt/error_handler_respects_current_error_reporting_level.phpt

```diff
@@ -21,14 +29,32 @@ while (!file_exists($vendor . '/vendor')) {
 
 require $vendor . '/vendor/autoload.php';
 
+$transportFactory = new class implements TransportFactoryInterface {
```

Comment:
Why are you adding this? If there was no need before there should be no need now :thinking:

Reply:
With the `dsn` option set, the `DefaultTransportFactory` no longer returns the `NullTransport` in this case, so there is actually a `Http\Discovery\NotFoundException` thrown during running this test.

Reply:
Makes sense, didn't think about this consequence 👍